scripts/deploy-cursorless-nvim.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# This script is used to push to the cursorless.nvim github production repo
+set -euo pipefail
+
+# Clone current cursorless.nvim main
+mkdir -p dist && cd dist
+git clone 'https://github.com/hands-free-vim/cursorless.nvim.git' cursorless.nvim-remote
+cd -
+
+out_dir=dist/cursorless.nvim-remote
+
+# Delete the old files
+cd "$out_dir"
+git rm -r README.md lua/ vim/ node/
+cd -
+
+#
+# Merge the build .js and the static files
+#
+
+# copy .lua, .vim dependencies, command-server and other static files
+cp -r cursorless.nvim/README.md "$out_dir/"
+cp -r cursorless.nvim/lua "$out_dir/"
+cp -r cursorless.nvim/vim "$out_dir/"
+cp -r cursorless.nvim/node "$out_dir/"
+
+# copy the built .js file
+mkdir -p "$out_dir/node/cursorless-neovim/out"
+cp packages/cursorless-neovim/package.json "$out_dir/node/cursorless-neovim/"
+cp packages/cursorless-neovim/out/index.cjs "$out_dir/node/cursorless-neovim/out/"
+
+# Push to cursorless.nvim
+cd "$out_dir"
+git add *
```

Comment:
I believe this should be `git add .`. Using `git add *` will skip hidden files

Reply:
shellcheck will complain about that too iirc, as you'd want git add -- *

Reply:
shellcheck doesn't complain about `git add .` on my machine. I'm pretty sure that's a fairly standard incantation, no?

Reply:
Sorry, I was agreeing with you. I meant it's another reason to not use `git add *`, as you need `git add -- *` for shellcheck to not complain, and yes `git add .` has no issues.

Reply:
ah right cool 👍